### ISSUE TITLE
Look into alternative storage methods

### ISSUE THREAD

**Issue by ominhas** [ominhas/workout-bot#6]
CSV or JSON are not the best ways to store our data. Look into other methods. (Possibility: TinyDB?)

**Comment by zainhussaini** [comment]
Yeah good point. I don't know database software, but I also saw sqlite3 recommended.
